docs/configuration/exporting/custom-webhooks.rst

```diff
@@ -0,0 +1,59 @@
+Custom Webhooks
+===============
+
+Send alerts from any monitoring system to Robusta via HTTP webhooks.
+
+.. note::
+    This feature requires Robusta Pro (SaaS or self-hosted commercial plans).
+
+Overview
+--------
+
+Forward alerts from any system that can send HTTP POST requests. Robusta will automatically enrich these alerts with Kubernetes context and apply your automation rules.
+
+Webhook Endpoint
+----------------
+
+Send alerts to Robusta using the following endpoint:
+
+.. code-block:: bash
+
+    POST https://api.robusta.dev/api/alerts
+
+Authentication
+--------------
+
+You'll need your API key and account ID:
+
+1. **Account ID**: Found in your ``generated_values.yaml`` file
+2. **API Key**: Generate this in the Robusta platform under **Settings** → **API Keys** → **New API Key**
+
+For detailed API documentation including request format, authentication, and examples, see :doc:`Alert History Import and Export API <exporting-data>`.
+
+Quick Example
+-------------
+
+Here's a simple example of sending a custom alert:
+
+.. code-block:: bash
```

Comment:
Please test as part of the review!

Reply:
Tested works 
<img width="2348" height="554" alt="CleanShot 2025-07-30 at 10 48 07@2x" src="https://github.com/user-attachments/assets/31ff6b24-008a-48c6-af12-81a29a8d16c0" />